<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvaluationListWindow</class>
 <widget class="QMainWindow" name="EvaluationListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liste des Évaluations</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="text">
       <string>Gestion des Évaluations</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutSearch">
      <item>
       <widget class="QLabel" name="lblSearch">
        <property name="text">
         <string>Recherche:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSearch">
        <property name="placeholderText">
         <string>Rechercher une évaluation...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFilter">
        <property name="text">
         <string>Filtrer par:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxFilter">
        <item>
         <property name="text">
          <string>Tous</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>En cours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Terminée</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutSort">
      <item>
       <widget class="QPushButton" name="btnSortDate">
        <property name="text">
         <string>Trier par Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSortNote">
        <property name="text">
         <string>Trier par Note</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnRefresh">
        <property name="text">
         <string>Actualiser</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableViewEvaluations">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutButtons">
      <item>
       <widget class="QPushButton" name="btnCreate">
        <property name="text">
         <string>Créer</string>
        </property>
        <property name="minimumHeight">
         <number>35</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Modifier</string>
        </property>
        <property name="minimumHeight">
         <number>35</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Supprimer</string>
        </property>
        <property name="minimumHeight">
         <number>35</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnExportPDF">
        <property name="text">
         <string>Export PDF</string>
        </property>
        <property name="minimumHeight">
         <number>35</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
